### PR TITLE
Change default value

### PR DESCRIPTION
**Pull request by eikeniebuhr-KAM**

### CODE REVIEW

File: release.php [hunk 1]
```diff
@@ -6,6 +6,7 @@
 $factory = new Factory();
 $consoleWriter = $factory->createConsoleWriter();
 $consoleQuestions = $factory->createConsoleQuestions();
+$minor = $factory->createMinor($tags);
```
Comment (ghost) [inline release.php]: $tags ??? This is not needed.

File: src/Sebastian301082/Console/Minor.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            if ($tagNumbers[0] == $highestMajorNumber) {
+                $major = strval($highestMajorNumber);
+                $minor = $tagNumbers[1] + 1;
+            }
+        }
+        $result = $major . '.' . $minor . '.' . 0;
+
+        return $result;
+    }
+}
```
Comment (ghost) [inline src/Sebastian301082/Console/Minor.php]: New line missing.

File: src/Sebastian301082/Factory.php [hunk 3]
```diff
@@ -35,4 +36,12 @@ private function createConsoleReader()
     {
         return new ConsoleReader();
     }
+
+    /**
+     * @return Minor
+     */
+    public function createMinor($tags)
```
Comment (ghost) [inline src/Sebastian301082/Factory.php]: $tags?? Not used.

File: src/Sebastian301082/Factory.php [hunk 4]
```diff
@@ -35,4 +36,12 @@ private function createConsoleReader()
...
+    {
+        return new Minor($tags);
```
Comment (ghost) [inline src/Sebastian301082/Factory.php]: $tags?? Not used.

File: release.php [hunk 5]
```diff
@@ -47,29 +48,30 @@
...
-        if(empty($matches)) {
+        if (empty($matches)) {
             continue;
         }
 
         $explodedTag = explode('.', $matches[0]);
-        $tags[$explodedTag[0]][$explodedTag[1]][$explodedTag[2]] = $matches[0];
+        $tagNumber = $explodedTag[0] . '.' . $explodedTag[1] . '.' . $explodedTag[2];
+
+        $tagsArray = array_push($tags, $tagNumber);
```
Comment (ghost) [inline release.php]: $tagsArray = not needed because it is never used.
BTW: array_push. What is this method returning?

File: release.php [hunk 6]
```diff
@@ -47,29 +48,30 @@
...
     }
 
     if (!empty($tags)) {
-        $major = count($tags) - 1;
-        $minor = count($tags[$major]);
-        $bugfix = 0;
-        $tag = "$major.$minor.$bugfix";
+        $default = $minor->minorPlusOne($tags);
+        $tag = "$default";
```
Comment (ghost) [inline release.php]: What's this line for?

**Comment by ghost** [comment]
Your PR looks really good. Some small clean up stuff is not there yet. Let us go through it step by step. For more details please have a look in my comments.

**Comment by eikeniebuhr-KAM** [comment]
Hey! Thanks, i changed some lines and now its working in more a OOP way, I will make a new PR (or update this one here).

**Comment by ghost** [comment]
Sounds Good. Looking forward to see your changes.

**Pull request closed without merging**